custom-namespaces.yaml

```diff
@@ -714,3 +714,26 @@ shared:
       type: table_view
       tables:
         - table: mozdata.static.data_incidents_v1
+moz_org_sites:
```

Comment:
Make this `websites`. Sometimes we have sites that are not exactly mozilla.org. Plus when marketers talk about moz org they are specifically referring to mozilla.org. e.g. Addons = addons.mozilla.org which isn't technically referred to when people thing moz org.